README.md

```
@@ -33,3 +33,116 @@
+작업자 서비스                             worker                      [C#],F#     Common/Worker/Web                 
+콘솔 앱                                   console                     [C#],F#,VB  Common/Console                    
+클래스 라이브러리                         classlib                    [C#],F#,VB  Common/Library                    
+프로토콜 버퍼 파일                        proto                                   Web/gRPC
+```
+
+
+``` zsh
+dotnet new <템플릿> -n <프로젝트이름>
+```
+
+### 이런 방법도 존재한다
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)